engine/physics/src/physics/physics_3d.cpp

```diff
@@ -181,6 +197,69 @@ namespace dmPhysics
                 return btCollisionWorld::ClosestRayResultCallback::addSingleResult(rayResult, normalInWorldSpace);
         }
 
+        void*           m_IgnoredUserData;
+        RayCastResponse m_Response;
+    };
+
+    // Grabbed from a more recent Bullet version for now
+    /// BULLET (do not modify) ->
+    struct AllHitsRayResultCallback : public btCollisionWorld::RayResultCallback
+    {
+        AllHitsRayResultCallback(const btVector3& rayFromWorld, const btVector3& rayToWorld)
+            : m_rayFromWorld(rayFromWorld)
+            , m_rayToWorld(rayToWorld)
+        {
+        }
+        btAlignedObjectArray<const btCollisionObject*> m_collisionObjects;
+        btAlignedObjectArray<btVector3> m_hitNormalWorld;
+        btAlignedObjectArray<btVector3> m_hitPointWorld;
+        btAlignedObjectArray<btScalar> m_hitFractions;
+        btVector3 m_rayFromWorld;//used to calculate hitPointWorld from hitFraction
+        btVector3 m_rayToWorld;
+
+        virtual btScalar addSingleResult(btCollisionWorld::LocalRayResult& rayResult, bool normalInWorldSpace)
+        {
+                m_collisionObject = rayResult.m_collisionObject;
+                m_collisionObjects.push_back(rayResult.m_collisionObject);
+                btVector3 hitNormalWorld;
+                if (normalInWorldSpace)
+                {
+                    hitNormalWorld = rayResult.m_hitNormalLocal;
+                } else
+                {
+                    hitNormalWorld = m_collisionObject->getWorldTransform().getBasis()*rayResult.m_hitNormalLocal;
+                }
+                m_hitNormalWorld.push_back(hitNormalWorld);
+                btVector3 hitPointWorld;
+                hitPointWorld.setInterpolate3(m_rayFromWorld,m_rayToWorld,rayResult.m_hitFraction);
+                m_hitPointWorld.push_back(hitPointWorld);
+                m_hitFractions.push_back(rayResult.m_hitFraction);
+                return m_closestHitFraction;
+        }
+    };
+    /// <- END BULLET
```

Comment:
We're currently using an older version of bullet, so I had to copy this class over.